modules/tow-boot/config.nix

```diff
@@ -119,6 +119,10 @@ in
       TPL_ENV_IS_NOWHERE = option yes;
       SPL_ENV_IS_NOWHERE = option yes;
     }))
+    (mkIf (variant == "boot-installer") (helpers: with helpers; {
+      TOW_BOOT_PREDICTABLE_BOOT_PREFER_EXTERNAL = yes;
+      TOW_BOOT_PREDICTABLE_BOOT_PREFER_INTERNAL = no;
```

Comment:
Since this is a `choice`, just the `PREFER_EXTERNAL = yes;` should be enough, but having a `no` for the other *ensures* there's nothing wrong going on with the implementation. It's redundant, but gives assurance both won't be true at once.